dss/util/auth/auth0.py

```diff
@@ -33,23 +33,21 @@ class Auth0AuthZGroupsMixin(Authorize):
     (Note: the Auth0 AuthZ extension adds groups, roles, and permissions,
     but here we just use groups.)
     """
-    def _get_auth0authz_claim(self):
+    @classmethod
+    def get_auth0authz_claim(self):
         oidc_audience = Config.get_audience()[0]
         return f"{oidc_audience}auth0"
```

Comment:
we might want to use a dedicated variable that is set in the env-var here, it would be easier for an operator, and less likely for them to question whats going on... there would need to be a change to the data-store-auth... which i just made, and came back to add into this comment. https://github.com/DataBiosphere/data-store-auth/pull/12

Reply:
👍  will update this to use `OIDC_AUTH0_TOKEN_CLAIM`

Reply:
Take this you might need it on your quest. https://github.com/DataBiosphere/data-store/pull/140

Reply:
Done in a7524c4

Reply:
the claim does not exist in `data-store/master` `environment` file.  #140 will add it in for you, ill let you merge it into this PR if you want :)

Reply:
👍  good call

Reply:
Done in 7e8ad0b